src/eventmodel.cpp

```diff
@@ -59,27 +59,27 @@ EventModel::~EventModel()
 QHash<int, QByteArray> EventModel::roleNames() const
 {
     QHash<int,QByteArray> roles;
```

Comment:
This is missing `ContactsRole`.

Reply:
Nicely spotted. Also at least EventRole. I'll readjust these to match the enum declaration order so it's easier to keep track.